app/models/bookmark.rb

```diff
@@ -0,0 +1,6 @@
+class Bookmark < ActiveRecord::Base
+  belongs_to :topic
+
+  validates :url, presence: true
```

Comment:
yes-revised to validate ``` url ``` for uniqueness and format.  Use the following to validate formate:  
``` URL_REGEX = /\A((https?:\/\/)|(www.))[\w]+\.com\z/i ```

I seemed to remember that we briefly discussed another way to validate format instead of through REGEX. Am I imagining that?